### PR TITLE
Feature/rx bloc list

### PR DESCRIPTION
**Pull request by DDavidPrime**
Add new package: rx_bloc_list

### CODE REVIEW

File: packages/flutter_rx_bloc/pubspec.yaml [hunk 1]
```diff
@@ -12,7 +12,8 @@ dependencies:
   flutter:
     sdk: flutter
   provider: ^5.0.0
-  rx_bloc: ^2.0.0
+  rx_bloc:
```
Comment (StanevPrime) [inline packages/flutter_rx_bloc/pubspec.yaml:15]: @DDavidPrime, I believe we need to have a specific version of rx_bloc

File: packages/rx_bloc_list/README.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
+## RxPaginatedBuilder
+
+*RxPaginatedBuilder* package giving the user the possibility to quickly add infinity scroll and pull-to-refresh features to their project with minimal setup. It provides the flexibility and simplicity of presentation of paginated data with the use of [RxBloc](https://github.com/Prime-Holding/rx_bloc "RxBloc")s inside the *RxBloc ecosystem*.
```
Comment (StanevPrime) [inline packages/rx_bloc_list/README.md:3]: @DDavidPrime, can we change the link from github to pub.dev ?

File: packages/rx_bloc_list/pubspec.yaml [hunk 3]
```diff
@@ -0,0 +1,37 @@
+name: rx_bloc_list
+description: A new Flutter application.
```
Comment (StanevPrime) [inline packages/rx_bloc_list/pubspec.yaml:2]: @DDavidPrime, could you please fill the description

File: packages/rx_bloc_list/pubspec.yaml [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+# The following line prevents the package from being accidentally published to
+# pub.dev using `pub publish`. This is preferred for private packages.
+publish_to: 'none' # Remove this line if you wish to publish to pub.dev
```
Comment (StanevPrime) [inline packages/rx_bloc_list/pubspec.yaml:6]: @DDavidPrime, also you can comment out publish_to

**Review by StanevPrime — CHANGES_REQUESTED** [review]

File: packages/flutter_rx_bloc/pubspec.yaml (hunk 1 above)
Comment (DDavidPrime) [reply packages/flutter_rx_bloc/pubspec.yaml]: Updated!

File: packages/rx_bloc_list/pubspec.yaml (hunk 3 above)
Comment (DDavidPrime) [reply packages/rx_bloc_list/pubspec.yaml]: Fixed!

File: packages/rx_bloc_list/pubspec.yaml (hunk 4 above)
Comment (DDavidPrime) [reply packages/rx_bloc_list/pubspec.yaml]: Fixed!

**Review by StanevPrime — APPROVED** [review]

**Pull request merged**